rebasehelper/build_helper.py

```diff
@@ -244,35 +243,27 @@ def get_detached_task(cls, task_id, results_dir):
         """
         raise NotImplementedError()
 
-    @classmethod
-    def _do_build_srpm(cls, spec, workdir, results_dir):
-        """
-        Build SRPM using rpmbuild.
-
-        :param spec: abs path to SPEC file inside the rpmbuild/SPECS in workdir.
-        :param workdir: abs path to working directory with rpmbuild directory
-                        structure, which will be used as HOME dir.
-        :param results_dir: abs path to dir where the log should be placed.
-        :return: If build process ends successfully returns abs path
-                 to built SRPM, otherwise 'None'.
-        """
-        logger.info("Building SRPM")
-        spec_loc, spec_name = os.path.split(spec)
-        output = os.path.join(results_dir, "build.log")
-
-        cmd = ['rpmbuild', '-bs', spec_name]
-        ret = ProcessHelper.run_subprocess_cwd_env(cmd,
-                                                   cwd=spec_loc,
-                                                   env={'HOME': workdir},
-                                                   output=output)
-
-        if ret != 0:
-            return None
-        else:
-            return PathHelper.find_first_file(workdir, '*.src.rpm')
+
+    @staticmethod
+    def get_builder_options(**kwargs):
+        builder_options = kwargs.get('builder_options', None)
```

Comment:
Using `None` as second argument here is redundant, see https://docs.python.org/3/library/stdtypes.html#dict.get.